htdp-lib/2htdp/private/check-aux.rkt

```diff
@@ -8,7 +8,14 @@
 
 (provide (all-defined-out))
 
-(define INSET  5)     ;; the space around the image in the canvas
+(define universe-inset  ;; the space around the image in the canvas
+  (make-parameter
+   5
+   (λ (v)
+     (unless (and (integer? v)
+                  (<= 0 v 1000))
+       (raise-argument-error 'universe-inset "an integer between 0 and 1000 (inclusive)" v))
+     v)))
```

Comment:
Whatever you really choose for an `inset` (1) name the max constant  and (2) use the name instead of the constant.